src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx

```diff
@@ -86,21 +87,20 @@ export const WalletMoreInfoCategory = ({
               >
                 <ListIcon as={FeatureIcon}></ListIcon>
 
-                <Text px={1} lineHeight={1}>
+                <Text as="span" whiteSpace="nowrap" px={1} lineHeight={1}>
```

Comment:
I don't think we want to use `nowrap` here...

<img width="1321" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/8758344e-0888-4f3c-9134-0f1a4094b83e">

It causes overflow.

This was why I had suggested breaking out the last word and attaching the icon to just that word... so the icon will never wrap by itself, but the words still will. Unfortunately there is no CSS pseudo-selector for ":last-word", which would make this easier. 

Hm.. thinking more here though, perhaps we could use a non-breaking-space `&nbsp;` between the string and the Emoji instead? Haven't tried yet, but could be worth a shot... should prevent the emoji from wrapping alone while allowing the spaces between the words to break a line.

Reply:
I think using `&nbsp;` ("non-breaking space") would work actually, @Dharmik79 are you down to give this a try?

Reply:
@wackerow The solution using &nbsp worked.